### ISSUE TITLE
Parameter subsets

### ISSUE THREAD

**Issue by ThorstenDP** [DroidPlanner/Tower#1586]
Hi all,

I just saw that https://github.com/DroidPlanner/Tower/issues/1580 was closed.
https://github.com/DroidPlanner/Tower/issues/1563 is related.

What would be the prefect is if a user could create specific and multiple lists for his own needs:
- sets for different copters
- sets for different batteries
- sets for different tuning parameters
- sets for different failsaves
- sets for different gimbals
etc.

Maybe as simple *.txt/*.xml files users can create from the full parameters list.

TIA and best regards,
Thorsten

**Comment by Thalek** [comment]
I agree.  While they are not a necessity, they would be an incredible convenience, saving considerable time for those fliers who have multiple devices with multiple battery types, and sometimes, a wide-ranging set of mission types.

Saving those parameters in a human-readable format is less efficient than a specialized binary format, but better for trouble-shooting, editing, and even off-line creation on another device.  (Some simple idiot-checking will prevent a badly formatted file from being used; with hand-editing, there'll always be at least one bad file out there.)

(I'm still looking forward to the day when I can do useful beta-testing instead of simply chiming in "me too!" on someone else's post.  The Pocket Drone fiasco has left me without a usable UAV for the near future.  My efforts to teach myself programming are not going well, either.)
